src/scf-helper-release/src/acceptance-tests-brain/test-resources/resources.csv

```diff
@@ -1 +1,2 @@
 cf-plugin-usb-linux-amd64.zip,https://github.com/SUSE/cf-usb-plugin/releases/download/0.0.1/cf-plugin-usb-linux-amd64.zip
+cf-plugin-backup-1.0.4+14.ged41cbc.linux-amd64.tgz,https://s3.amazonaws.com/cf-opensusefs2/cf-plugin-backup/cf-plugin-backup-1.0.4%2b14.ged41cbc.linux-amd64.tgz
```

Comment:
The `ed41cbc` commit is the plugin master from before the plugin fixes done by https://github.com/SUSE/cf-plugin-backup/pull/51
Do we have a check build whose url we could insert here for testing ?

Reply:
That's independent of this – the plugin works (with silly messages) without that PR.

Reply:
Ok. Going forward with testing.